ansible_collections/arista/avd/roles/eos_designs/defaults/main.yml

```diff
@@ -204,3 +112,81 @@ connected_endpoints_keys:
 #custom_structured_configuration_prefix allows the user to customize the prefix for Custom Structured Configuration variables.
 custom_structured_configuration_prefix: custom_structured_configuration_
 custom_structured_configuration_list_merge: 'replace'
+
+switch_type_keys:
```

Comment:
I think this would be more readable if we explicitly set all variables, and not just rely on defaults.
Alternatively, they could be set but just commented out with a `#`

Reply:
I had that at first, and it was _not_ very readable. We don't add other unset/default vars and this file is not read by regular users.
![image](https://user-images.githubusercontent.com/48674677/123741592-9f162280-d8aa-11eb-8100-0e730fe114db.png)

If the documentation is clear, I think that should serve as guide.